app/pages/capability-selector/capabilitySelectorPageContext.js

```diff
@@ -0,0 +1,20 @@
+import content from './manifest.json';
+
+export const createCapabilitySelectorPageContext = ({ capabilities }) => (
+  {
+    ...content,
+    capabilities: capabilities.capabilities.reduce((acc, capability, i) => {
```

Comment:
I'm not capable of handling so many capabilities in one line. Is there a way to tidy this up a little please?

Reply:
of course